.github/workflows/tf-account.yaml

```diff
@@ -42,13 +48,36 @@ jobs:
           role-session-name: ${{ github.sha }}
           aws-region: ${{ secrets.AWS_REGION }}
 
+      - name: Get Version Changed
+        id: changed-versions
+        uses: tj-actions/changed-files@v25
+        with:
+          files: versions.tf
```

Comment:
It is possible for a provider upgrade to be available even if `versions.tf` was not modified, e.g:
```  
required_providers {
    aws = {
      version = "> 4.0"
    }
```

We don't want users to have to always bump things up manually.

Reply:
Any change to the file should trigger it. Adding or removing a newline at the end of the file should allow the users to bump things arbitrarily.